router/presentation/query.go

```diff
@@ -27,8 +27,9 @@ func GetTiemRange(values url.Values) (start time.Time, end time.Time, err error)
 type UserRelation int
 
 const (
-	RelationBelongs = iota
-	RelationAdmins  = iota
+	RelationBelongs         = iota
+	RelationAdmins          = iota
+	RelationBelongsOrAdmins = iota
```

Comment:
```suggestion
	RelationBelongs         UserRelation = iota
	RelationAdmins          
	RelationBelongsOrAdmins
```
かも？今のままだとuntyped intとかになってそう